velox/expression/FunctionSignature.cpp

```diff
@@ -139,6 +139,48 @@ TypeSignature parseTypeSignature(const std::string& signature) {
   return TypeSignature(baseName, std::move(nestedTypes));
 }
 
+TypePtr signatureToVeloxType(const TypeSignature& typeSignature) {
+  const auto& baseName = typeSignature.baseName();
+  if (baseName == "tinyint") {
```

Comment:
Can we use mapNameToTypeKind here?

Reply:
It seems that the `kTypeStringMap` does not contain *decimal*.